examples/multi_slice.py

```diff
@@ -6,74 +6,95 @@
 renderer = gfx.renderers.WgpuRenderer(canvas)
 scene = gfx.Scene()
 
 background = gfx.Background(gfx.BackgroundMaterial((0, 1, 0, 1), (0, 1, 1, 1)))
 scene.add(background)
 
-scene.add(gfx.AxesHelper(size=10))
+scene.add(gfx.AxesHelper(size=50))
 
 vol = imageio.volread("imageio:stent.npz")
 tex = gfx.Texture(vol, dim=3, usage="sampled")
 view = tex.get_view(filter="linear")
 material = gfx.MeshVolumeSliceMaterial(map=view, clim=(0, 255))
 
+# TODO: also add a mesh slice for each plane
+
 for axis in [0, 1, 2]:
-    nslices = vol.shape[0]
-
-    # TODO: if we could set the slicing plane on the material parametrically
-    # we could reuse the same plane geometry for all slices here
-    # TODO: there seems to be a problem with the texcoords... only one plane looks correct
-    # TODO: why is the third axis not in [0..1] range?
-    # TODO: also add a mesh slice for each plane
-    geometry = gfx.PlaneGeometry(200, 200, 1, 1)
     if axis == 0:  # YZ plane
+        geometry = gfx.PlaneGeometry(vol.shape[1], vol.shape[0], 1, 1)
         texcoords = np.array(
-            [[0.5, 0.0, 0], [0.5, 1.0, 0], [0.5, 0.0, nslices], [0.5, 1.0, nslices],],
-            dtype=np.float32,
+            [[0.5, 0, 0], [0.5, 1, 0], [0.5, 0, 1], [0.5, 1, 1]], dtype=np.float32,
         )
     elif axis == 1:  # XZ plane
+        geometry = gfx.PlaneGeometry(vol.shape[2], vol.shape[0], 1, 1)
         texcoords = np.array(
-            [[0.0, 0.5, 0], [1.0, 0.5, 0], [0.0, 0.5, nslices], [1.0, 0.5, nslices],],
-            dtype=np.float32,
+            [[0, 0.5, 0], [1, 0.5, 0], [0, 0.5, 1], [1, 0.5, 1]], dtype=np.float32,
         )
     elif axis == 2:  # XY plane (default)
-        z_idx = nslices / 2
+        geometry = gfx.PlaneGeometry(vol.shape[2], vol.shape[1], 1, 1)
         texcoords = np.array(
-            [
-                [0.0, 0.0, z_idx],
-                [1.0, 0.0, z_idx],
-                [0.0, 1.0, z_idx],
-                [1.0, 1.0, z_idx],
-            ],
-            dtype=np.float32,
+            [[0, 0, 0.5], [1, 0, 0.5], [0, 1, 0.5], [1, 1, 0.5]], dtype=np.float32,
         )
-    print(texcoords)
     geometry.texcoords = gfx.Buffer(texcoords, usage="vertex|storage")
 
     plane = gfx.Mesh(geometry, material)
 
     # by default the plane is in XY plane
     if axis == 0:  # YZ plane
-        plane.rotation.set_from_axis_angle(gfx.linalg.Vector3(0, 1, 0), 0.5 * np.pi)
+        plane.rotation.set_from_euler(gfx.linalg.Euler(0.5 * np.pi, 0.5 * np.pi))
     elif axis == 1:  # XZ plane
-        plane.rotation.set_from_axis_angle(gfx.linalg.Vector3(1, 0, 0), 0.5 * np.pi)
+        plane.rotation.set_from_euler(gfx.linalg.Euler(0.5 * np.pi))
     scene.add(plane)
 
 camera = gfx.OrthographicCamera(200, 200)
 camera.position.set(50, 50, 50)
 camera.look_at(gfx.linalg.Vector3())
+controls = gfx.OrbitControls(camera.position.clone(), up=gfx.linalg.Vector3(0, 0, 1))
 
 
 def animate():
+    # todo: should the control set it directly?
```

Comment:
no, then the user would lose control over the control flow

Reply:
I kind of had to violate my own policy in the zoom_to_point code.. don't like it. :/ I'll see if I can implement it without having to mutate the camera